deb/build/debian/control

```diff
@@ -2,7 +2,7 @@ Source: @@ARTIFACTNAME@@
 Section: devel
 Priority: extra
 Maintainer: @@AUTHOR@@
-Build-Depends: debhelper (>= 9)
+Build-Depends: debhelper (>= 9), dh-systemd
```

Comment:
Given Jenkins 2.54 requires Java 8 which is not available on Ubuntu Trusty 14.04, I guess that can be discarded. The point stand for other .deb based distributions that might not have systemd.